cairo/tests/utils/serde.py

```diff
@@ -61,10 +62,16 @@ def get_struct_definition(program, struct_name):
 
 
 class Serde:
-    def __init__(self, segments, program):
+    def __init__(self, segments, program, cairo_file):
         self.segments = segments
         self.memory = segments.memory
         self.program = program
+        self.cairo_file = cairo_file
+
+    @property
+    def main_part(self):
```

Comment:
ik this is only to go fast etc but it's a bit hard to follow the purpose of these functions without a one-liner comment